### DISCUSSION TITLE
New migration tool

### DISCUSSION THREAD

**Discussion by jklm2828** [iotaledger/firefly#8034] [Help & Support]
Hello,

i wanted to import my IOTA from trinity to firefly. I know that the foundation is working on a new migration tool that it will be possible to import the IOTA with trinity seed or a kdbx seedvault file. Do you guys know when this will be ready? Can you give a time frame for it maybe? 

Thanks for helping! 

Best Regards

**Comment by begonaalvarezd** [comment]
🔈 **We just released the new legacy migration tool** 🔈

Read about it here: https://blog.iota.org/iota-legacy-migration-tool/ 

If you face any issues please contact us over discord https://discord.iota.org/ or open an issue directly in the legacy tool repo https://github.com/iotaledger/legacy-migration-tool

Happy migration 🌷

**Comment by GeLa1005** [comment]
Hi there, 

I have the same problem.
I can't transfer, or cannot find a possibilty toi transfer my IOTA from Trinity to firefly. 


What should I do next?

**Reply by begonaalvarezd** [reply]
I added a tutorial to the messge right above https://github.com/iotaledger/firefly/discussions/8034#discussioncomment-8559565 did you follow the tutorial? You dont have to move funds from trinity to firefly, you have to migrate
